clmm/cosmology/ccl.py

```diff
@@ -28,25 +28,32 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
         # this tag will be used to check if the cosmology object is accepted by the modeling
-        self.backend = 'ccl'
+        self.backend = "ccl"
 
         assert isinstance(self.be_cosmo, ccl.Cosmology)
 
         # cor factor for sigma_critical
-        self.cor_factor = _patch_rho_crit_to_cd2018(
-            ccl.physical_constants.RHO_CRITICAL)
+        self.cor_factor = _patch_rho_crit_to_cd2018(ccl.physical_constants.RHO_CRITICAL)
 
     def _init_from_cosmo(self, be_cosmo):
-
         assert isinstance(be_cosmo, ccl.Cosmology)
         self.be_cosmo = be_cosmo
 
     def _init_from_params(self, H0, Omega_b0, Omega_dm0, Omega_k0):
-
+        # pylint: disable=arguments-differ
```

Comment:
Is this commented line needed?

Reply:
yes, they are to tell pylint yo skip a specific check at a specific location. There are some cases where we do want to have code that differs from pylint's formatting, like having arguments with Upper and Lower case (`Omega_dm0`) and with less than three characters (`H0`)